src/plugins/core/filters.ts

```diff
@@ -353,6 +353,16 @@ export class FiltersPlugin extends CorePlugin<FiltersState> implements FiltersSt
   }
 
   exportForExcel(data: ExcelWorkbookData) {
-    this.export(data);
+    for (const sheet of data.sheets) {
+      for (const filterTable of this.getFilterTables(sheet.id)) {
+        if (zoneToDimension(filterTable.zone).height === 1) {
+          continue;
+        }
+        sheet.filterTables.push({
+          range: zoneToXc(filterTable.zone),
+          filters: [],
```

Comment:
The code in `export` (that was used before this commit) doesn't export `filters: [],`. Any reason to export that here ? Is it necessary ?

Reply:
it's a typing issue. we were calling `export` which expects a `WorkbookData`, so the absence of filter was not noticed. I guess typescript cannot find this kind of issue because we modify in place, with no return type.